api/controllers/tokens.js

```diff
@@ -4,10 +4,11 @@ const TokenGenerator = require("../models/token_generator")
 const SessionsController = {
 
   Create: (req, res) => {
-    const email = req.body.email;
+    // const email = req.body.email;
+    const username = req.body.username;
     const password = req.body.password;
 
-    User.findOne({ email: email }).then(async (user) => {
+    User.findOne({ username: username }).then(async (user) => {
```

Comment:
Great work - venturing into the darkest corners of the code base ..!

Reply:
😁 just following the breadcrumbs.